website/content/en/docs/helm/reference/advanced_features.md

```diff
@@ -91,7 +93,20 @@ Depending on the number of CRs of the same type, a single reconciling worker may
 For example:
 
 ```sh
-$ operator-sdk exec-entrypoint helm --max-workers 10
+$ cat config/manager/manager.yaml 
+...
+    spec:
+      containers:
+      - args:
+        - manager
+        - --max-workers=10
```

Comment:
Let's go ahead and document the new name. We also need to rewrite line 91 now that the default value is `runtime.NumCPU()`.


Also this might need to be specified in the manager's patch file in ./config/default/ for it to be applied.

```suggestion
        - --max-concurrent-reconciles=10
```